# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/mc/roff-no-sgr.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/mc.ext.in.orig	2000-07-19 09:25:48.000000000 +0300
+++ ./lib/mc.ext.in	2003-06-24 19:20:50.000000000 +0300
@@ -348,26 +348,26 @@
 
 # Manual page
 regex/(([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|n)|\.man)$
-	Open=nroff @TROFFASCII@ @MANDOC@ %f | %var{PAGER:more}
-	View=%view{ascii,nroff} nroff @TROFFASCII@ @MANDOC@ %f
+	Open=GROFF_NO_SGR= nroff @TROFFASCII@ @MANDOC@ %f | %var{PAGER:more}
+	View=%view{ascii,nroff} GROFF_NO_SGR= nroff @TROFFASCII@ @MANDOC@ %f
 	Icon=man.xpm
 
 # Troff with me macros.
 shell/.me
-	Open=nroff @TROFFASCII@ -me %f | %var{PAGER:more}
-	View=%view{ascii,nroff} nroff @TROFFASCII@ -me %f
+	Open=GROFF_NO_SGR= nroff @TROFFASCII@ -me %f | %var{PAGER:more}
+	View=%view{ascii,nroff} GROFF_NO_SGR= nroff @TROFFASCII@ -me %f
 	Icon=man.xpm
 
 # Troff with ms macros.
 shell/.ms
-	Open=nroff @TROFFASCII@ -ms %f | %var{PAGER:more}
-	View=%view{ascii,nroff} nroff @TROFFASCII@ -ms %f
+	Open=GROFF_NO_SGR= nroff @TROFFASCII@ -ms %f | %var{PAGER:more}
+	View=%view{ascii,nroff} GROFF_NO_SGR= nroff @TROFFASCII@ -ms %f
 	Icon=man.xpm
 
 # Manual page - compressed
 regex/([^0-9]|^[^\.]*)\.([1-9][a-z]?|n)\.g?[Zz]$
-	Open=gunzip -dc %f | nroff @TROFFASCII@ @MANDOC@ | %var{PAGER:more}
-	View=%view{ascii,nroff} gunzip -dc %f | nroff @TROFFASCII@ @MANDOC@
+	Open=gunzip -dc %f | GROFF_NO_SGR= nroff @TROFFASCII@ @MANDOC@ | %var{PAGER:more}
+	View=%view{ascii,nroff} gunzip -dc %f | GROFF_NO_SGR= nroff @TROFFASCII@ @MANDOC@
 	Icon=man.xpm
 
 regex/(read\.?me$)|(README)
